DOCS-2427: Add Duopact documentation (#1489)
MultiSafepay/docs: content/business/accounting/integrations/Yuki/integrations/duopact.md

---
title: "Duopact"
weight: 10
meta_title: "Yuki integrations - Duopact - MultiSafepay Docs"
meta_description: "Sign up. Build and test your payments integration. Explore our products and services. Use our API Reference, SDKs, and wrappers. Get support."
logo: '/svgs/Duopact.png'
layout: 'single'
read_more: "."
title_short: "Yuki reconciliation by Duopact"
short_description: "Automatically upload your MultiSafepay transactions to Yuki"
extra_button:
    url: "https://www.snelkoppeling.eu/productoverzicht/webwinkelkoppelingen"
    text: "Go to site"
    type: "primary"
---

> You can integrate with Duopact using your [MultiSafepay API key](/tools/multisafepay-control/retrieving-your-api-key/). 

Reviewer: ```suggestion
You can integrate with Duopact using your [MultiSafepay API key](/getting-started/set-up-your-account/user-guide/viewing-your-api-key/). 
```

## Connecting Duopact with MultiSafepay

> To connect Duopact with your MultiSafepay account, [contact duopact](https://www.duopact.nl/nl/contact/). Provide Duopact with your MultiSafepay API key. They will set up a Yuki account for you. 

Reviewer: ```suggestion
To connect Duopact with your MultiSafepay account, see Duopact – [Contact](https://www.duopact.nl/nl/contact/). Provide Duopact with your MultiSafepay API key. They will set up a Yuki account for you. 
```

## Using Duopact with MultiSafepay

MultiSafepay transactions are automatically imported daily at midnight.

To manually import transactions:

> 1. Log in to your [Duopact account](portal.yukiconnector.nl).

Reviewer: ```suggestion
1. Sign in to your [Duopact account](portal.yukiconnector.nl).
```

> 2. If you operate multiple domains, select the relevant domain from the top-left menu.

Reviewer: ```suggestion
2. If you operate multiple websites, select the relevant site from the top-left menu.
```

3. Go to **Bankmutaties** > **MultiSafepay**.
4. Click the green button under the **Status** tab.

> Manually importing transactions doesn't affect automatic importing.

Reviewer: ```suggestion
**Note:** Manually importing transactions doesn't affect automatic importing.
```
